sql/hive/v0.13.1/src/main/scala/org/apache/spark/sql/hive/Shim13.scala

```diff
@@ -218,7 +218,13 @@ private[hive] object HiveShim {
       TypeInfoFactory.voidTypeInfo, null)
 
   def getStringWritable(value: Any): hadoopIo.Text =
-    if (value == null) null else new hadoopIo.Text(value.asInstanceOf[UTF8String].toString)
```

Comment:
Let's keep it unchanged, since we can pass in the `UTF8String` instead of `String`, the same in `Shim12.scala`